tests/voq/test_voq_nbr.py

```diff
@@ -701,7 +902,11 @@ def localport_adminup(self, dut, dut_intf):
 
         """
         logger.info("Admin up port %s/%s", dut.hostname, dut_intf)
-        dut.no_shutdown(dut_intf)
+        if dut.is_multi_asic:
```

Comment:
we can use `asic.startup()` for single and multi asic

Reply:
Fixed in latest.